backend/apis/invoices/get-invoice-pdf.api.js

```diff
@@ -237,7 +237,7 @@ app.get("/api/invoices/:invoiceId/pdfs", (req, res) => {
           );
 
           const amount = `$${
-            invoice.totalCharged ? invoice.totalCharged.toFixed(2) : "0.00"
+            invoice.totalCharged ? invoice.totalCharged : "0.00"
```

Comment:
In `mysql2` package, a column with `decimal(15, 2)` comes back as a string rather than a number